src/azure-cli/azure/cli/command_modules/acs/_roleassignments.py

```diff
@@ -0,0 +1,168 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
This file is basically the same as the [_roleassignments.py](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/_roleassignments.py) file in aks-preview,  except that the error type is changed to `AzCLIError` and add backward compatibility for creating role assignment.